db/queries.py

```diff
@@ -3,44 +3,90 @@
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    queryset = Message.objects.filter(
+        text__icontains=word
+    ).select_related("user", "chat")
+
+    return list(queryset)
 
 
 def get_untitled_chats() -> list[Chat]:
-    pass
+    return list(Chat.objects.filter(title__startswith="Untitled"))
 
 
-def get_users_who_sent_messages_in_2015() -> list[str]:
-    pass
+def get_users_who_sent_messages_in_2015() -> list[Message]:
```

Comment:
incorrect return type
[list[tuple[str]]](https://stackoverflow.com/questions/37205793/django-values-list-vs-values#:~:text=The%20values_list()%20method%20returns%20a%20QuerySet%20containing%20tuples%3A)